public/template/layout.jade

```diff
@@ -10,13 +10,15 @@ html(lang="en")
 
       link(href="/stylesheet/bootstrap.min.css", rel="stylesheet")
       link(href="/stylesheet/bootstrap-responsive.min.css", rel="stylesheet")
+      link(href="/stylesheet/font-awesome.min.css", rel="stylesheet")
       link(href="/stylesheet/fineuploader.css", rel="stylesheet")
       link(href="/stylesheet/select2.css", rel="stylesheet")
 
     else
 
       link(href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.3.2/css/bootstrap.min.css", rel="stylesheet", integrity="sha384-MY3lqAyZWEjvYefTFTBYUN85pnmwAxdSjsX+8IqCP1N5VBzOIX6GNo1rUt9gz74C", crossorigin="anonymous")
       link(href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.3.2/css/bootstrap-responsive.min.css", rel="stylesheet", integrity="sha384-F6SkVapMeW8omBv9cyqV+15oQ6VSZ+g9B3dXePPOKKVGEdFOfLo50I4anpVkG3kQ", crossorigin="anonymous")
+      link(href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css", rel="stylesheet", integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN", crossorigin="anonymous")
```

Comment:
Where was Glyphicons originally being included from? We should remove that reference for perf